### PR TITLE
Fix/security 2020 06

### PR DESCRIPTION
**Pull request by lewisdaly**
- minor tidy of `readme.md`
- ignore unfixable devDependencies for 1 month
- bump hapi version to `19.1.1`
- make `audit:check` only run on production dependencies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,23 +17,23 @@
...
         "boom": "7.3.0",
-        "@hapi/hapi": "18.4.0",
         "hapi-openapi": "1.2.4"
     },
     "devDependencies": {
         "faucet": "0.0.1",
-        "npm-audit-resolver": "^2.1.0",
-        "npm-check-updates": "3.2.1",
-        "nyc": "14.1.1",
+        "npm-audit-resolver": "^2.2.0",
```
Comment (rmothilal) [inline package.json]: caret on dependency

File: package.json [hunk 2]
```diff
@@ -17,23 +17,23 @@
...
+        "npm-check-updates": "7.0.1",
+        "nyc": "15.1.0",
         "pre-commit": "1.2.2",
         "proxyquire": "2.1.3",
-        "sinon": "7.5.0",
-        "standard": "^14.3.1",
+        "sinon": "9.0.2",
+        "standard": "^14.3.4",
```
Comment (rmothilal) [inline package.json]: caret on dependency

File: package.json [hunk 3]
```diff
@@ -17,23 +17,23 @@
         "registry": "https://registry.npmjs.org"
     },
     "dependencies": {
-        "@mojaloop/central-services-logger": "8.1.2",
+        "@hapi/hapi": "18.4.1",
+        "@mojaloop/central-services-logger": "10.4.0",
         "boom": "7.3.0",
-        "@hapi/hapi": "18.4.0",
         "hapi-openapi": "1.2.4"
```
Comment (rmothilal) [inline package.json]: hapi-openapi can be updated to the latest version and so can hapi. this should get rid of your audit issues

**Review by rmothilal — CHANGES_REQUESTED** [review]
Please see comment

File: package.json (hunk 1 above)
Comment (lewisdaly) [reply package.json]: I don't think we should worry about these in `devDependencies`

Comment (rmothilal) [reply package.json]: i do think for consistency it should be removed, will remove any confusion

**Comment by lewisdaly** [comment]
@rmothilal I tried updating to the latest hapi, but this broke our unit tests, so I didn't want to touch it. 

Specifically 4 tests looking at the response status code failed, they were expecting `200`s, but actually got `204`s.

**Comment by lewisdaly** [comment]
Hapi update will be covered in: https://github.com/mojaloop/project/issues/1460

**Review by rmothilal — APPROVED** [review]
Happy

**Pull request merged**